hw/ip/otbn/data/base-insns.yml

```diff
@@ -356,34 +356,63 @@
 - mnemonic: csrrs
   rv32i: true
   synopsis: Atomic Read and Set bits in CSR
-  operands: [grd, csr, grs]
+  operands: [grd, csr, grs1]
+  doc: |
+    Reads the value of the CSR `csr`, and writes it to the destination GPR `grd`.
+    The initial value in `grs1` is treated as a bit mask that specifies bit positions to be set in the CSR.
```

Comment:
Maybe "... that specifies bits to be set ..."? I think I'd find that easier to understand